js_modules/dagit/packages/core/src/runs/RunTableNew.tsx

```diff
@@ -240,8 +256,26 @@ const RunRow: React.FC<{
 
   const isReexecution = run.tags.some((tag) => tag.key === DagsterTag.ParentRunId);
 
+  const [showRunTags, setShowRunTags] = React.useState(false);
+  const [showRunTagsIcon, setShowRunTagsIcon] = React.useState(false);
+
+  const containerRef = React.useRef<HTMLTableRowElement | null>(null);
+  React.useLayoutEffect(() => {
+    const mousemoveListener = (ev: MouseEvent) => {
+      if (!containerRef.current?.contains(ev.target as HTMLElement)) {
+        setShowRunTagsIcon(false);
+      } else {
+        setShowRunTagsIcon(true);
+      }
+    };
+    document.addEventListener('mousemove', mousemoveListener);
```

Comment:
`mousemove` seems a bit more aggressive than necessary -- can you use `onMouseEnter` on the row instead?

Reply:
I needed a mouseout listener anyways so I used mousemove. I did originally use onMouseEnter on the `tr` but for some reason it wasn't firing so I switched to mousemove. I can try it again maybe I was doing something wrong before?